src/utils/profile.ts

```diff
@@ -19,5 +20,21 @@ export const updateProfileEnteredVenueIds = async (
   }
 };
 
+export const getUserDisplayStatus = (
+  user?: WithId<User>
+): RecentUserStatusType => {
+  if (user?.status) return RecentUserStatusType.busy;
+
+  return user ? RecentUserStatusType.online : RecentUserStatusType.offline;
+};
```

Comment:
The logic here seems weird/wrong?

When `user?.status` is true we'll never get to the 2nd line.

Reply:
Now it makes sense. fixed in 5a3581bc4abfb14880e2e621a12c0994e01cace8